core-bundle/src/Resources/contao/library/Contao/Validator.php

```diff
@@ -254,7 +254,7 @@ public static function isUuid($varValue)
 	/**
 	 * Valid binary UUID (version 1)
 	 *
-	 * @param mixed $varValue The value to be validated
+	 * @param string $varValue The value to be validated
```

Comment:
Did you intentionally only fix this one in the `Validator``class?

Nearly all of them only accept strings because they use string functions (like `strlen`). At least under PHP>=8 this produces warnings/errors if not called with a string.

Reply:
AFAIK `strlen` can be used with all scalar types without notice or warning in PHP 8.

Reply:
Was it the preg functions then? 🤔 I remember running across this recently in some PR.

Reply:
At least for objects the error message states it must be a string. 🙃

> strlen(): Argument #1 ($str) must be of type string

Reply:
I only changed it there because the code inspector complained about it. However, I agree that the change is wrong and I have fixed that in 444c3b1277afa488b92e7774128f0cbbacc64c9e.